### PR TITLE
Do not render past events link if there are no past events.

### LINKED ISSUE #21762: [BUG] Broken link is created by VAMC Past Events listing page when no past events exist in the CMS
## Problem description

If a VAMC region has no past events, the 'Past Events' link is still created and shown on [the page](https://www.va.gov/eastern-oklahoma-health-care/events/):

Screen Shot 2021-03-19 at 9 55 34 AM

This generates a broken link error, because the past events page itself is not generated, and fails the build.

## How to reproduce

In the CMS, remove all past events for a given VAMC.

## Suggested resolution

The build process should be modified to either
- check to ensure that the Past Events link is only generated if there is a past events page, or
- always generate the past events page with placeholder text if no events are available

See [Slack thread ](https://dsva.slack.com/archives/CBU0KDSB1/p1616165909204100)

### PR DESCRIPTION
**Pull request by mpelzsherman**
Resolves https://github.com/department-of-veterans-affairs/va.gov-team/issues/21762

### CODE REVIEW

File: src/site/stages/build/plugins/download-drupal-assets.js [hunk 1]
```diff
@@ -16,6 +16,9 @@ async function downloadFile(
   everythingDownloaded,
 ) {
   const asset = assetsToDownload.shift();
+  if (!asset) {
+    return;
+  }
```
Comment (mpelzsherman) [inline src/site/stages/build/plugins/download-drupal-assets.js:21]: Apparently, undefined objects can end up in `assetsToDownload` somehow?

Comment (ncksllvn) [reply src/site/stages/build/plugins/download-drupal-assets.js:21]: No idea, how did you find this?

**Review by ncksllvn — APPROVED** [review]
LGTM

**Review by brangi — APPROVED** [review]

**Review by alexandec — APPROVED** [review]

File: src/site/stages/build/plugins/download-drupal-assets.js (hunk 1 above)
Comment (mpelzsherman) [reply src/site/stages/build/plugins/download-drupal-assets.js:21]: My build broke, and this fixed it.

**Pull request merged**